services-custom/dynamodb-enhanced/src/test/java/software/amazon/awssdk/enhanced/dynamodb/functionaltests/UpdateBehaviorTest.java

```diff
@@ -141,4 +145,29 @@ public void updateBehaviors_transactWriteItems_secondUpdate() {
         assertThat(persistedRecord.getLastAutoUpdatedOn()).isAfterOrEqualTo(INSTANT_2);
         assertThat(persistedRecord.getCreatedAutoUpdateOn()).isEqualTo(firstUpdatedRecord.getCreatedAutoUpdateOn());
     }
+
+    /**
+     * Currently, nested records are not updated through extensions.
```

Comment:
Added this test case for future changes to behavior of nested records with tagged attributes that are manipulated through extensions and operations.